src/components/filters/input.tsx

```diff
@@ -5,7 +5,7 @@ import Input from "../input/index"
 import { scrollIntoViewIfMobile } from "../../lib/scrollHelper"
 import getClosestElement from "../../lib/getClosestElement"
 
-interface Props {
+export interface Props {
```

Comment:
What's the motivation for exporting props?

Reply:
thanks for asking - not obvious at all. the updated storybook dependencies expect an interface Props to be exported (I guess to generate prop docs?) - I don't recall what tooling, but something was yelling at me about it

Reply:
Okay, that makes sense. I suppose that might be then tricky to integrate with the index export, but worth trying still.